app/javascript/application.js

```diff
@@ -0,0 +1,31 @@
+// Entry point for the build script in your package.json
+// import '@hotwired/turbo-rails';
+// import './controllers';
```

Comment:
- [ ] [REQUIRED] Good job so far! Please clear unnecessary comments in your codebase after that, your work will be even more clean and safe😃